src/pages/index.js

```diff
@@ -1,10 +1,52 @@
 import React from "react";
-
+import { useStaticQuery, graphql } from "gatsby";
 import Layout from "../components/layout";
 import SEO from "../components/seo";
 import heroImage from "../images/lukas-schulz-n6uOlqYPMXY-unsplash.jpg";
+import RandomRouteCard from "../components/RandomRouteCard";
+
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  //The maximum is exclusive and the minimum is inclusive
+  return Math.floor(Math.random() * (max - min) + min); 
+}
 
 function IndexPage() {
+  let allClimbingRoutes = useStaticQuery(graphql`
+  query {
+    allMdx(
+          filter:{fields: { collection: { eq: "climbing-routes"}}}
+        ) {
+          edges {
+            node {
+              fields {
+                parentId
+                slug
+              }
+              frontmatter {
+                route_name
+                yds
+                type {
+                  tr
+                  trad
+                  sport
+                  boulder
+                }
+                metadata {
+                  legacy_id
+                }
+              }
+            }
+          }
+        }
+    }
+  `);
+  const min = 0;
+  const max = allClimbingRoutes.allMdx.edges.length;
+  const randomIndex = getRandomInt(min,max);
+  const randomClimb = allClimbingRoutes.allMdx.edges[randomIndex].node;
```

Comment:
Once we have a larger dataset perhaps we could generate a list of say 20 random climbs at build time, and at runtime we cycle through that list.